torch/csrc/cuda/nccl.cpp

```diff
@@ -625,7 +629,7 @@ void all_gather(
 #endif
 }
 
-void all2all(at::Tensor& input,
+void all2all_single_equal_split(at::Tensor& input,
```

Comment:
Would I be correct if I assume this API is not visible to users?

Reply:
It seems so? I can not find anything about nccl at https://pytorch.org/cppdocs/api/library_root.html